Milestone 4 - Add Media Queries

t-bello7 opened the pull request:
- Add media queries to the portfolio project
- Style the Desktop View

t-bello7 committed:
style:
- add media query for desktop
- change flex-direction to row for affected elements
- add menu list in navigation
- add menu-btn icon
- style menu

t-bello7 committed:
fix:
- change top padding of home first child

t-bello7 committed:
fix :
- fix hr line for mobile and full width
- fix background issue on about me
- fix background position on about me

t-bello7 committed:
refac:
- add desktop view images

t-bello7 committed:
style: -style home desktop view
- add grid-container
- arrange items of grid elements
- change flex direction of grid item contact link

t-bello7 committed:
style: - add padding and margin to container in desktop view
- add hr to work section head
- add display flex to work sectin header

t-bello7 committed:
style:
- add width of half  width view to about-bio
- add space-between to skill
- align skill to flex-start

t-bello7 committed:
style: change column number for project container for tab view

t-bello7 committed:
fix: fix stylelint issue

gemmen29 commented on index.html:
```diff
@@ -156,9 +179,11 @@ <h3 class="text-light ff-inai">
-      <div class="contact-form mg-8 ">
+
+    <footer> 
+      <div class="contact-form container mg-8 flex">
```
- [x] Kindly increase the width of the contact form and make the `Get in touch` button align the inputs from the beginning(not in the center) to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/165759287-285a1255-3745-4905-bcb4-a71cedf4f3d5.png) | ![image](https://user-images.githubusercontent.com/24703171/165759070-907e9b97-b85d-4103-9477-7a7faee1de32.png) |

gemmen29 reviewed (changes requested):
Hi @t-bello7, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 

## Required Changes ♻️
_Check the comments under the review._

![](https://media.giphy.com/media/tKyrc0NTlEcGWdThp4/giphy.gif)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

t-bello7 committed:
refac: add flex-basis to contact form giving it a constant width
- align contact form to stretch
- align contact button to start
- fix linter issues

t-bello7 committed:
- delete one h file

OmarRamoun reviewed (approved):
# 🟢 Project Approved! 🏆

## Hi @t-bello7 👋

Your project is complete! ✔️ 🏁

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ descriptive PR title and description
- ✔️✔️ very descriptive commit messages
- ✔️ implementation similar to design

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️
- This is My **GitHub**: [![@mrramoun](https://img.shields.io/github/followers/MrRamoun?label=Ramoun&style=social)](https://github.com/mrramoun)
- This is My **Twitter**: [![@ramoun16](https://img.shields.io/twitter/follow/ramoun16?label=ramoun16&style=social)](https://twitter.com/ramoun16)
- This is My **Slack**: [![Ramoun](https://img.shields.io/badge/slack-@Ramoun-brightgreen)](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
